extensions/amp-base-carousel/1.0/scroller.js

```diff
@@ -210,7 +207,14 @@ export {Scroller};
  * @param {!BaseCarouselDef.SlideProps} props
  * @return {PreactDef.Renderable}
  */
-function renderSlides({children, restingIndex, offsetRef, pivotIndex, loop}) {
+function renderSlides({
+  children,
+  restingIndex,
+  offsetRef,
+  pivotIndex,
+  loop,
+  styles,
```

Comment:
This is not a part of `BaseCarouselDef.SlideProps`. So I think it's better if you pass it as a second arg.